hack/tools.go

```diff
@@ -0,0 +1,38 @@
+// +build tools
+
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tools
+
+// This package imports things required by this repository, to force `go mod` to see them as dependencies
+import (
```

Comment:
I think only adding `_ "knative.dev/pkg/hack"` would suffice. Could you try removing all other and see if it'll work?

Reply:
looks like we don't need code-generator related imports. I think we still need knative.dev/test-infra/scripts https://github.com/knative/net-certmanager/blob/master/hack/release.sh#L20